lib/song.rb

```diff
@@ -25,9 +22,11 @@ def artist=(song_artist)
 
   def self.new_from_filename(filename)
     song_instance = filename.gsub(".mp3", "").split(' - ')
-    new(song_instance[1],
```

Comment:
Use correct indentation. Each argument should be on it's own line. Brackets should also be on their own lines